rollup.config.ts

```diff
@@ -32,54 +42,7 @@ const config = ({ input, output }) => ({
     // Resolve source maps to the original source
     sourceMaps(),
   ],
-})
-export default [
-  config({
-    input: `src/${libraryName}.ts`,
-    name: _.camelCase(libraryName),
-    output: [
-      {
-        file: pkg.main,
-        name: 'votes',
-        format: 'umd',
-        sourcemap: true,
-        globals: {
-          lodash: '_',
-        },
-      },
-      {
-        file: pkg.module,
-        name: 'votes',
-        format: 'es',
-        sourcemap: true,
-        globals: {
-          lodash: '_',
-        },
-      },
-    ],
-  }),
-  config({
-    input: 'src/utils/index.ts',
-    name: 'utils',
-    output: [
-      {
-        file: 'dist/utils.umd.js',
-        name: 'utils',
-        format: 'umd',
-        sourcemap: true,
-        globals: {
-          lodash: '_',
-        },
-      },
-      {
-        file: 'dist/utils.es5.js',
-        name: 'utils',
-        format: 'es',
-        sourcemap: true,
-        globals: {
-          lodash: '_',
-        },
-      },
-    ],
-  }),
-]
+  watch: { include: 'src/**' },
+}
+
+export default rollupConfig
```

Comment:
Codacy found an issue: [Use of default exports is forbidden](https://app.codacy.com/gh/lzear/votes/pullRequest?prid=6374053)